types/pixi/placeableObjects/drawing.d.ts

```diff
@@ -0,0 +1,297 @@
+declare class Drawing extends PlaceableObject<Drawing.Data> {
+  /**
+   * The inner drawing container
+   */
+  drawing: PIXI.Container | null;
+
+  /**
+   * The primary drawing shape
+   */
+  shape: PIXI.Graphics | null;
+
+  /**
+   * Text content, if included
+   */
+  text: PIXI.Text | null;
+
+  /**
+   * The Graphics outer frame and handles
+   */
+  frame: PIXI.Container | null;
+
+  /**
+   * Internal timestamp for the previous freehand draw time, to limit sampling
+   */
+  protected _drawTime: number;
+  protected _sampleTime: number;
+
+  /**
+   * Internal flag for the permanent points of the polygon
+   */
+  protected _fixedPoints: Array<[number, number]>;
+
+  /** @override */
+  static get embeddedName(): 'Drawing';
+
+  /* -------------------------------------------- */
+  /*  Properties                                  */
+  /* -------------------------------------------- */
+
+  /**
+   * @remarks
+   * Not implemented for Drawing
+   */
+  get bounds(): never;
```

Comment:
I'm not sure if this is the best way to go about this for this method. Based on a grep of foundry.js, only two of the `PlaceableObject`s implement `get bounds`: `AmbientLight` and `Wall`.

Reply:
I think we can do it like that, yeah.